css/style.css

```diff
@@ -0,0 +1,84 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 48px;
+  width: 100%;
+  margin: 0;
+}
+
+.logo-container {
+  box-sizing: border-box;
+  height: 48px;
+  padding: 10px 12px 10px 12px;
+  display: flex;
+}
+
+.text-logo {
+  color: #28352f;
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.nav-menu {
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.headline {
+  background-image: url(/images/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 648px;
+}
+
+.main-titel {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.secondary-titel {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #36b37e;
+}
+
+.des-text-container {
+  width: 327px;
+  height: 164px;
+  margin: 30px;
+}
+
+.description-text {

```

Comment:
- [ ] I would suggest updating the color for this text to match the template color: :100: 

![image](https://user-images.githubusercontent.com/81522435/138139279-888e5cda-a282-4339-a7d0-17105b8a1e30.png)